### PR TITLE
Add support for Sidekiq 6.5.7

### PR DESCRIPTION
**Pull request by sensorsasha**
Update all the needed dependencies and pick the right version to make it work on Sidekiq 6.5.7.

* Move to Sidekiq 6.5
* Move to sidekiq-throttled 0.18.0
* Drop redis namespaces support
* Fix deprecation warnings for redis gem

### CODE REVIEW

**Comment by sensorsasha** [comment]
@ixti we're moving closer to the latest sidekiq-throttled release 🥲

**Review by kevinrobell-st — APPROVED** [review]

File: lib/sidekiq/ultimate/fetch.rb [hunk 1]
```diff
@@ -45,12 +46,15 @@ def retrieve_work
         work
       end
 
-      def self.bulk_requeue(units, _options)
+      # TODO: Requeue in batch or at least using pipeline
```
Comment (davidbodow-st) [inline lib/sidekiq/ultimate/fetch.rb:49]: Let's log an issue for this in the repo and link here

**Review by davidbodow-st — APPROVED** [review]

File: lib/sidekiq/ultimate/fetch.rb (hunk 1 above)
Comment (sensorsasha) [reply lib/sidekiq/ultimate/fetch.rb:49]: I'll be touching this anyway in the following PR for the 7.2 migration.

Comment (davidbodow-st) [reply lib/sidekiq/ultimate/fetch.rb:49]: Feel free to merge then

**Pull request merged**